dask/dataframe/methods.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function, absolute_import, division
```

Comment:
Unsure if this separate file makes sense anymore, with using `methodcaller` objects now. One slight benefit is it moves some of the trivial chunk/aggregate functions out of the `core.py` file, which has become a bit bloated.